action-api/src/main/java/com/powsybl/action/AbstractLoadActionBuilder.java

```diff
@@ -39,4 +54,21 @@ public B withReactivePowerValue(double reactivePowerValue) {
         this.reactivePowerValue = reactivePowerValue;
         return (B) this;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        AbstractLoadActionBuilder<?, ?> that = (AbstractLoadActionBuilder<?, ?>) o;
+        return Objects.equals(id, that.id) && Objects.equals(elementId, that.elementId) && Objects.equals(relativeValue, that.relativeValue) && Objects.equals(activePowerValue, that.activePowerValue) && Objects.equals(reactivePowerValue, that.reactivePowerValue);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, elementId, relativeValue, activePowerValue, reactivePowerValue);
```

Comment:
I would add the class name as parameter of Objects.hash to make the hashcode even more specific (in case you have different builders that get the same values even for different variables)